MicroPython/src/peripherals/sensor.py

```diff
@@ -0,0 +1,110 @@
+import dht
+import utime
+import machine
+import logging
+import lib.bme280 as bme280
+
+TIME_TO_DHT11_TO_WAKE_UP_S = 1
+TIME_TO_DHT22_TO_WAKE_UP_S = 0.5
+TIME_TO_BME280_TO_WAKE_UP_S = 0.5
+
+FAILED_TO_MEASURE_VALUE = -99
+
+
+class Sensor:
```

Comment:
Is this class fully responsible for DHT sensors and in case of BME works as a wrapper?

Reply:
Class is fully responsible for all types of sensors